keyboards/bastardkb/tbk/config.h

```diff
@@ -0,0 +1,39 @@
+/* 
+ * Copyright 2021 Quentin LEBASTARD <qlebastard@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#include "config_common.h"
+#define VENDOR_ID 0xA8F8
+#define PRODUCT_ID 0x1828
+#define DEVICE_VER 0x0001
+#define MANUFACTURER Bastard Keyboards
+#define PRODUCT The Bastard Keyboard
+
+#define MATRIX_ROWS 10
+#define MATRIX_COLS 6
+#define DIODE_DIRECTION ROW2COL
+#define MATRIX_ROW_PINS { D7, B5, F7, F6, B6 }
+#define MATRIX_COL_PINS { B4, E6, C6, B1, B3, B2 }
+#define RGB_DI_PIN D2
+#define RGBLED_NUM 38
+#define RGBLIGHT_SPLIT
+#define RGBLIGHT_ANIMATIONS
+#define DEBOUNCE 5
+#define SOFT_SERIAL_PIN D0
+#define LOCKING_SUPPORT_ENABLE
+#define LOCKING_RESYNC_ENABLE
+#define MASTER_RIGHT
```

Comment:
Hi, what better way do you have in mind ? I'm not super familiar with the locking options

Reply:
Oh, yeah there's no point of having the locking there, the PCBs don't support Alps switches. Let me remove them

Reply:
No, specifically, the `#define MASTER_LEFT`.  For instance, xyverz would actually need to undefine this, to get the `EE_HANDS` option working.

Also, not having a master left/right or ee hands define defaults to `MASTER_LEFT`.

If you really want it to default to master right, you could add a `post_config.h` file, and fill it with: 
```c
#pragma once

#if !defined(EE_HANDS) && !defined(MASTER_RIGHT)
#    define MASTER_RIGHT
#endif
```

Reply:
Hi,

I pre-flash my Elite-Cs before shipping, so needing to have different options on each one will bring confusion during the build (the right one might end up in the left and vice-versa)...

I will look at the post_config, thanks !

Reply:
Well, the master right means that that whatever side is plugged in, both are treated as right side.  EE Hands (or even split pin) would be better, as no matter which side is plugged in, it is handled correctly.  Both are a bit more work, but are a better solution, IMO. 

For reference: https://docs.qmk.fm/#/feature_split_keyboard?id=setting-handedness

But this should be fine.  Worst case, somebody has to add `#undef MASTER_RIGHT` to change this. So not a huge deal.

Reply:
Hi, I looked into it. From skim reading it looks like it would require the user to add an additional cable (to set a pin to high), and I want to keep it as simple as possible. 
So at the moment I'm keeping master_right